shell/common/tracing_controller.cc

```diff
@@ -30,6 +30,8 @@ static void AddTraceMetadata() {
   blink::Threads::Gpu()->PostTask([]() { Dart_SetThreadName("gpu_thread"); });
   blink::Threads::UI()->PostTask([]() { Dart_SetThreadName("ui_thread"); });
   blink::Threads::IO()->PostTask([]() { Dart_SetThreadName("io_thread"); });
+  blink::Threads::Platform()->PostTask(
```

Comment:
Flutter doesn't create the platform thread, so I'd prefer to keep its original name.

Also, it looks like the platform thread and the UI thread map to the same thread on Fuchsia